### PR TITLE
Feature/link text to image

### PR DESCRIPTION
**Pull request by giacomocerre**

### CODE REVIEW

File: src/app/panels/image-panel/image-panel.component.ts [hunk 1]
```diff
@@ -22,9 +36,16 @@ export class ImagePanelComponent {
 
   constructor(
     private evtModelService: EVTModelService,
+    private evtStatus: EVTStatusService,
   ) {
   }
 
+  updatePage(viewerPage: number){
+    this.evtModelService.pages$.pipe(take(1)).subscribe(
```
Comment (szenzaro) [inline src/app/panels/image-panel/image-panel.component.ts:44]: Create a new subscription at every page change creates a memory leak.
Ensure there is a single subscription (e.g. inside the constructor) and chain the event stream of the osd-component to an observable property of this component.

**Review by szenzaro — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Updated to Angular 9
 
 ### Added
+- Link between text and images
```
Comment (ChiaraDipi) [inline CHANGELOG.md:13]: This message is a little bit misleading: it seems that you have implemented the Image Text Linking feature. 
Change into something like "Text/images connection in page change"

File: src/app/panels/image-panel/image-panel.component.ts [hunk 3]
```diff
@@ -1,17 +1,31 @@
 import { Component, Input } from '@angular/core';
 import { BehaviorSubject, combineLatest } from 'rxjs';
-import { filter, map } from 'rxjs/operators';
-import { ViewerDataType } from '../../models/evt-models';
+import { filter, map, take } from 'rxjs/operators';
+import { EVTStatusService } from 'src/app/services/evt-status.service';
```
Comment (ChiaraDipi) [inline src/app/panels/image-panel/image-panel.component.ts:4]: Use relative path

File: src/app/panels/image-panel/image-panel.component.ts (hunk 1 above)
Comment (ChiaraDipi) [reply src/app/panels/image-panel/image-panel.component.ts:44]: @szenzaro Can you suggest the better way of doing it?

**Review by ChiaraDipi — CHANGES_REQUESTED** [review]

File: src/app/panels/image-panel/image-panel.component.ts (hunk 1 above)
Comment (szenzaro) [reply src/app/panels/image-panel/image-panel.component.ts]: Check and review last commit

**Review by ChiaraDipi — APPROVED** [review]

**Review by szenzaro — APPROVED** [review]

**Pull request merged**